start-stop-mwaa-environment - npm build error

Hi team,

I strictly followed the instructions - having prerequisite prepared and exporting relevant env variables in shell, but I got this error when I ran ```npm build```:

```
lib/lambda/dags-cli.test.ts:49:7 - error TS2322: Type 'AwsStub<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>' is not assignable to type 'never'.

49       mwaaMock = mockClient(MWAAClient);
         ~~~~~~~~

lib/lambda/dags-cli.test.ts:49:29 - error TS2345: Argument of type 'typeof MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.
  Type 'typeof MWAAClient' is not assignable to type 'ClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.
    The types of 'prototype.middlewareStack.concat' are incompatible between these types.
      Type '<InputType extends ServiceInputTypes, OutputType extends ServiceOutputTypes>(from: MiddlewareStack<InputType, OutputType>) => MiddlewareStack<...>' is not assignable to type '<InputType extends ServiceInputTypes, OutputType extends MetadataBearer>(from: MiddlewareStack<InputType, OutputType>) => MiddlewareStack<InputType, OutputType>'.
        Types of parameters 'from' and 'from' are incompatible.
          Type 'MiddlewareStack<InputType, OutputType>' is not assignable to type 'MiddlewareStack<InputType, ServiceOutputTypes>'.
            Types of property 'addRelativeTo' are incompatible.
              Type '(middleware: MiddlewareType<InputType, OutputType>, options: RelativeMiddlewareOptions) => void' is not assignable to type '(middleware: MiddlewareType<InputType, ServiceOutputTypes>, options: RelativeMiddlewareOptions) => void'.
                Types of parameters 'middleware' and 'middleware' are incompatible.
                  Type 'MiddlewareType<InputType, ServiceOutputTypes>' is not assignable to type 'MiddlewareType<InputType, OutputType>'.
                    Type 'InitializeMiddleware<InputType, ServiceOutputTypes>' is not assignable to type 'MiddlewareType<InputType, OutputType>'.
                      Type 'InitializeMiddleware<InputType, ServiceOutputTypes>' is not assignable to type 'InitializeMiddleware<InputType, OutputType>'.
                        Call signature return types 'InitializeHandler<InputType, ServiceOutputTypes>' and 'InitializeHandler<InputType, OutputType>' are incompatible.
                          Type 'Promise<InitializeHandlerOutput<ServiceOutputTypes>>' is not assignable to type 'Promise<InitializeHandlerOutput<OutputType>>'.
                            Type 'InitializeHandlerOutput<ServiceOutputTypes>' is not assignable to type 'InitializeHandlerOutput<OutputType>'.
                              Types of property 'output' are incompatible.
                                Type 'ServiceOutputTypes' is not assignable to type 'OutputType'.
                                  'ServiceOutputTypes' is assignable to the constraint of type 'OutputType', but 'OutputType' could be instantiated with a different subtype of constraint 'MetadataBearer'.
                                    Type 'CreateCliTokenCommandOutput' is not assignable to type 'OutputType'.
                                      'CreateCliTokenCommandOutput' is assignable to the constraint of type 'OutputType', but 'OutputType' could be instantiated with a different subtype of constraint 'MetadataBearer'.

49       mwaaMock = mockClient(MWAAClient);
                               ~~~~~~~~~~

lib/lambda/dags-cli.test.ts:50:16 - error TS2339: Property 'on' does not exist on type 'never'.

50       mwaaMock.on(CreateCliTokenCommand).resolves(token);
                  ~~

lib/lambda/dags-cli.test.ts:54:16 - error TS2339: Property 'restore' does not exist on type 'never'.

54       mwaaMock.restore();
                  ~~~~~~~

lib/lambda/dags-cli.test.ts:75:48 - error TS2345: Argument of type 'typeof CreateCliTokenCommand' is not assignable to parameter of type 'new (input: CreateCliTokenCommandInput) => AwsCommand<CreateCliTokenCommandInput, MetadataBearer>'.
  Construct signature return types 'CreateCliTokenCommand' and 'AwsCommand<CreateCliTokenCommandInput, MetadataBearer>' are incompatible.
    The types of 'middlewareStack.concat' are incompatible between these types.
      Type '<InputType extends CreateCliTokenCommandInput, OutputType extends CreateCliTokenCommandOutput>(from: MiddlewareStack<InputType, OutputType>) => MiddlewareStack<...>' is not assignable to type '<InputType extends CreateCliTokenCommandInput, OutputType extends MetadataBearer>(from: MiddlewareStack<InputType, OutputType>) => MiddlewareStack<...>'.
        Types of parameters 'from' and 'from' are incompatible.
          Type 'MiddlewareStack<InputType, OutputType>' is not assignable to type 'MiddlewareStack<InputType, CreateCliTokenCommandOutput>'.
            Types of property 'addRelativeTo' are incompatible.
              Type '(middleware: MiddlewareType<InputType, OutputType>, options: RelativeMiddlewareOptions) => void' is not assignable to type '(middleware: MiddlewareType<InputType, CreateCliTokenCommandOutput>, options: RelativeMiddlewareOptions) => void'.
                Types of parameters 'middleware' and 'middleware' are incompatible.
                  Type 'MiddlewareType<InputType, CreateCliTokenCommandOutput>' is not assignable to type 'MiddlewareType<InputType, OutputType>'.
                    Type 'InitializeMiddleware<InputType, CreateCliTokenCommandOutput>' is not assignable to type 'MiddlewareType<InputType, OutputType>'.
                      Type 'InitializeMiddleware<InputType, CreateCliTokenCommandOutput>' is not assignable to type 'InitializeMiddleware<InputType, OutputType>'.
                        Call signature return types 'InitializeHandler<InputType, CreateCliTokenCommandOutput>' and 'InitializeHandler<InputType, OutputType>' are incompatible.
                          Type 'Promise<InitializeHandlerOutput<CreateCliTokenCommandOutput>>' is not assignable to type 'Promise<InitializeHandlerOutput<OutputType>>'.
                            Type 'InitializeHandlerOutput<CreateCliTokenCommandOutput>' is not assignable to type 'InitializeHandlerOutput<OutputType>'.
                              Types of property 'output' are incompatible.
                                Type 'CreateCliTokenCommandOutput' is not assignable to type 'OutputType'.
                                  'CreateCliTokenCommandOutput' is assignable to the constraint of type 'OutputType', but 'OutputType' could be instantiated with a different subtype of constraint 'MetadataBearer'.

75         expect(mwaaMock).toHaveReceivedCommand(CreateCliTokenCommand);
                                                  ~~~~~~~~~~~~~~~~~~~~~

lib/lambda/dags-cli.test.ts:85:53 - error TS2345: Argument of type 'typeof CreateCliTokenCommand' is not assignable to parameter of type 'new (input: CreateCliTokenCommandInput) => AwsCommand<CreateCliTokenCommandInput, MetadataBearer>'.
  Type 'CreateCliTokenCommand' is not assignable to type 'AwsCommand<CreateCliTokenCommandInput, MetadataBearer>'.

85         expect(mwaaMock).toHaveReceivedCommandTimes(CreateCliTokenCommand, 1);
                                                       ~~~~~~~~~~~~~~~~~~~~~

lib/lambda/mwaa-new-environment-function.test.ts:27:5 - error TS2322: Type 'AwsStub<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>' is not assignable to type 'never'.

27     mwaaMock = mockClient(MWAAClient);
       ~~~~~~~~

lib/lambda/mwaa-new-environment-function.test.ts:27:27 - error TS2345: Argument of type 'typeof MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

27     mwaaMock = mockClient(MWAAClient);
                             ~~~~~~~~~~

lib/lambda/mwaa-new-environment-function.test.ts:31:14 - error TS2339: Property 'restore' does not exist on type 'never'.

31     mwaaMock.restore();
                ~~~~~~~

lib/lambda/mwaa-new-environment-function.test.ts:35:14 - error TS2339: Property 'on' does not exist on type 'never'.

35     mwaaMock.on(CreateEnvironmentCommand).resolves({ Arn: 'my-env-arn' });
                ~~

lib/lambda/mwaa-new-environment-function.test.ts:48:44 - error TS2345: Argument of type 'typeof CreateEnvironmentCommand' is not assignable to parameter of type 'new (input: CreateEnvironmentCommandInput) => AwsCommand<CreateEnvironmentCommandInput, MetadataBearer>'.
  Construct signature return types 'CreateEnvironmentCommand' and 'AwsCommand<CreateEnvironmentCommandInput, MetadataBearer>' are incompatible.
    The types of 'middlewareStack.concat' are incompatible between these types.
      Type '<InputType extends CreateEnvironmentCommandInput, OutputType extends CreateEnvironmentCommandOutput>(from: MiddlewareStack<InputType, OutputType>) => MiddlewareStack<...>' is not assignable to type '<InputType extends CreateEnvironmentCommandInput, OutputType extends MetadataBearer>(from: MiddlewareStack<InputType, OutputType>) => MiddlewareStack<...>'.
        Types of parameters 'from' and 'from' are incompatible.
          Type 'MiddlewareStack<InputType, OutputType>' is not assignable to type 'MiddlewareStack<InputType, CreateEnvironmentCommandOutput>'.
            Types of property 'addRelativeTo' are incompatible.
              Type '(middleware: MiddlewareType<InputType, OutputType>, options: RelativeMiddlewareOptions) => void' is not assignable to type '(middleware: MiddlewareType<InputType, CreateEnvironmentCommandOutput>, options: RelativeMiddlewareOptions) => void'.
                Types of parameters 'middleware' and 'middleware' are incompatible.
                  Type 'MiddlewareType<InputType, CreateEnvironmentCommandOutput>' is not assignable to type 'MiddlewareType<InputType, OutputType>'.
                    Type 'InitializeMiddleware<InputType, CreateEnvironmentCommandOutput>' is not assignable to type 'MiddlewareType<InputType, OutputType>'.
                      Type 'InitializeMiddleware<InputType, CreateEnvironmentCommandOutput>' is not assignable to type 'InitializeMiddleware<InputType, OutputType>'.
                        Call signature return types 'InitializeHandler<InputType, CreateEnvironmentCommandOutput>' and 'InitializeHandler<InputType, OutputType>' are incompatible.
                          Type 'Promise<InitializeHandlerOutput<CreateEnvironmentCommandOutput>>' is not assignable to type 'Promise<InitializeHandlerOutput<OutputType>>'.
                            Type 'InitializeHandlerOutput<CreateEnvironmentCommandOutput>' is not assignable to type 'InitializeHandlerOutput<OutputType>'.
                              Types of property 'output' are incompatible.
                                Type 'CreateEnvironmentCommandOutput' is not assignable to type 'OutputType'.
                                  'CreateEnvironmentCommandOutput' is assignable to the constraint of type 'OutputType', but 'OutputType' could be instantiated with a different subtype of constraint 'MetadataBearer'.

48     expect(mwaaMock).toHaveReceivedCommand(CreateEnvironmentCommand);
                                              ~~~~~~~~~~~~~~~~~~~~~~~~

lib/lambda/mwaa-new-environment-function.test.ts:53:14 - error TS2339: Property 'on' does not exist on type 'never'.

53     mwaaMock.on(CreateEnvironmentCommand).resolves({ Arn: 'my-env-arn' });
                ~~

lib/lambda/mwaa-new-environment-function.test.ts:66:44 - error TS2345: Argument of type 'typeof CreateEnvironmentCommand' is not assignable to parameter of type 'new (input: CreateEnvironmentCommandInput) => AwsCommand<CreateEnvironmentCommandInput, MetadataBearer>'.

66     expect(mwaaMock).toHaveReceivedCommand(CreateEnvironmentCommand);
                                              ~~~~~~~~~~~~~~~~~~~~~~~~

lib/lambda/mwaa-new-environment-function.test.ts:71:14 - error TS2339: Property 'on' does not exist on type 'never'.

71     mwaaMock.on(CreateEnvironmentCommand).resolves({ Arn: 'my-env-arn' });
                ~~

lib/lambda/mwaa-new-environment-function.test.ts:83:44 - error TS2345: Argument of type 'typeof CreateEnvironmentCommand' is not assignable to parameter of type 'new (input: CreateEnvironmentCommandInput) => AwsCommand<CreateEnvironmentCommandInput, MetadataBearer>'.

83     expect(mwaaMock).toHaveReceivedCommand(CreateEnvironmentCommand);
                                              ~~~~~~~~~~~~~~~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:33:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.
  Type 'MWAAClient' is not assignable to type 'Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>'.

33       const mwaaClientMock = mockClient(mwaaClient);
                                           ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:35:9 - error TS2353: Object literal may only specify known properties, and 'Environment' does not exist in type 'CommandResponse<MetadataBearer>'.

35         Environment: {
           ~~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:48:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

48       const mwaaClientMock = mockClient(mwaaClient);
                                           ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:50:9 - error TS2353: Object literal may only specify known properties, and 'Environment' does not exist in type 'CommandResponse<MetadataBearer>'.

50         Environment: {
           ~~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:63:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

63       const mwaaClientMock = mockClient(mwaaClient);
                                           ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:74:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

74       const mwaaClientMock = mockClient(mwaaClient);
                                           ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:87:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

87       const mwaaClientMock = mockClient(mwaaClient);
                                           ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:102:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

102       const mwaaClientMock = mockClient(mwaaClient);
                                            ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:104:9 - error TS2353: Object literal may only specify known properties, and 'Environment' does not exist in type 'CommandResponse<MetadataBearer>'.

104         Environment: {
            ~~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:117:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

117       const mwaaClientMock = mockClient(mwaaClient);
                                            ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:119:9 - error TS2353: Object literal may only specify known properties, and 'Environment' does not exist in type 'CommandResponse<MetadataBearer>'.

119         Environment: {
            ~~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:132:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

132       const mwaaClientMock = mockClient(mwaaClient);
                                            ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:143:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

143       const mwaaClientMock = mockClient(mwaaClient);
                                            ~~~~~~~~~~

lib/lambda/mwaa-status-poller.test.ts:156:41 - error TS2345: Argument of type 'MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

156       const mwaaClientMock = mockClient(mwaaClient);
                                            ~~~~~~~~~~

lib/lambda/mwaa-update-environment-function.test.ts:27:5 - error TS2322: Type 'AwsStub<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>' is not assignable to type 'never'.

27     mwaaMock = mockClient(MWAAClient);
       ~~~~~~~~

lib/lambda/mwaa-update-environment-function.test.ts:27:27 - error TS2345: Argument of type 'typeof MWAAClient' is not assignable to parameter of type 'InstanceOrClassType<Client<ServiceInputTypes, MetadataBearer, SmithyResolvedConfiguration<HttpHandlerOptions>>>'.

27     mwaaMock = mockClient(MWAAClient);
                             ~~~~~~~~~~

lib/lambda/mwaa-update-environment-function.test.ts:31:14 - error TS2339: Property 'restore' does not exist on type 'never'.

31     mwaaMock.restore();
                ~~~~~~~

lib/lambda/mwaa-update-environment-function.test.ts:35:14 - error TS2339: Property 'on' does not exist on type 'never'.

35     mwaaMock.on(UpdateEnvironmentCommand).resolves({ Arn: 'my-env-arn' });
                ~~

lib/lambda/mwaa-update-environment-function.test.ts:39:44 - error TS2345: Argument of type 'typeof UpdateEnvironmentCommand' is not assignable to parameter of type 'new (input: UpdateEnvironmentCommandInput) => AwsCommand<UpdateEnvironmentCommandInput, MetadataBearer>'.
  Construct signature return types 'UpdateEnvironmentCommand' and 'AwsCommand<UpdateEnvironmentCommandInput, MetadataBearer>' are incompatible.
    The types of 'middlewareStack.concat' are incompatible between these types.
      Type '<InputType extends UpdateEnvironmentCommandInput, OutputType extends UpdateEnvironmentCommandOutput>(from: MiddlewareStack<InputType, OutputType>) => MiddlewareStack<...>' is not assignable to type '<InputType extends UpdateEnvironmentCommandInput, OutputType extends MetadataBearer>(from: MiddlewareStack<InputType, OutputType>) => MiddlewareStack<...>'.
        Types of parameters 'from' and 'from' are incompatible.
          Type 'MiddlewareStack<InputType, OutputType>' is not assignable to type 'MiddlewareStack<InputType, UpdateEnvironmentCommandOutput>'.
            Types of property 'addRelativeTo' are incompatible.
              Type '(middleware: MiddlewareType<InputType, OutputType>, options: RelativeMiddlewareOptions) => void' is not assignable to type '(middleware: MiddlewareType<InputType, UpdateEnvironmentCommandOutput>, options: RelativeMiddlewareOptions) => void'.
                Types of parameters 'middleware' and 'middleware' are incompatible.
                  Type 'MiddlewareType<InputType, UpdateEnvironmentCommandOutput>' is not assignable to type 'MiddlewareType<InputType, OutputType>'.
                    Type 'InitializeMiddleware<InputType, UpdateEnvironmentCommandOutput>' is not assignable to type 'MiddlewareType<InputType, OutputType>'.
                      Type 'InitializeMiddleware<InputType, UpdateEnvironmentCommandOutput>' is not assignable to type 'InitializeMiddleware<InputType, OutputType>'.
                        Call signature return types 'InitializeHandler<InputType, UpdateEnvironmentCommandOutput>' and 'InitializeHandler<InputType, OutputType>' are incompatible.
                          Type 'Promise<InitializeHandlerOutput<UpdateEnvironmentCommandOutput>>' is not assignable to type 'Promise<InitializeHandlerOutput<OutputType>>'.
                            Type 'InitializeHandlerOutput<UpdateEnvironmentCommandOutput>' is not assignable to type 'InitializeHandlerOutput<OutputType>'.
                              Types of property 'output' are incompatible.
                                Type 'UpdateEnvironmentCommandOutput' is not assignable to type 'OutputType'.
                                  'UpdateEnvironmentCommandOutput' is assignable to the constraint of type 'OutputType', but 'OutputType' could be instantiated with a different subtype of constraint 'MetadataBearer'.

39     expect(mwaaMock).toHaveReceivedCommand(UpdateEnvironmentCommand);
                                              ~~~~~~~~~~~~~~~~~~~~~~~~


Found 34 errors in 4 files.

Errors  Files
     6  lib/lambda/dags-cli.test.ts:49
     9  lib/lambda/mwaa-new-environment-function.test.ts:27
    14  lib/lambda/mwaa-status-poller.test.ts:33
     5  lib/lambda/mwaa-update-environment-function.test.ts:27
```

Could you please help to direct the right way to troubleshoot the issues above? Thank you!

--- a/README.md
+++ b/README.md
@@ -21,6 +21,7 @@ This [folder](./usecases) contains complete set of sample use cases including do
 
 * [Image Processing Pipeline](usecases/image-processing)
 * [Amazon MWAA with AWS CodeArtifact for Python dependencies](usecases/mwaa-with-codeartifact)
+* [Stop and Start MWAA](usecases/start-stop-mwaa-environment/)
 
 ### Requirements.txt
 

--- a/usecases/README.md
+++ b/usecases/README.md
@@ -7,3 +7,4 @@ Complete set of sample use cases including documentation, infrastructure as code
 3. [Amazon MWAA metadata database migration](metadata-migration)
 4. [Amazon MWAA custom domain for public webserver](mwaa-public-webserver-custom-domain)
 5. [Amazon MWAA in private VPC with Amazon Cognito](mwaa-cognito-cdk)
+6. [Stop and Start MWAA](start-stop-mwaa-environment)

--- a/usecases/start-stop-mwaa-environment/package.json
+++ b/usecases/start-stop-mwaa-environment/package.json
@@ -29,9 +29,9 @@
     "@types/uuid": "9.0.8",
     "@typescript-eslint/eslint-plugin": "6.4.0",
     "@typescript-eslint/parser": "6.4.0",
-    "aws-cdk": "2.139.0",
-    "aws-sdk-client-mock": "4.0.0",
-    "aws-sdk-client-mock-jest": "4.0.0",
+    "aws-cdk": "2.147.3",
+    "aws-sdk-client-mock": "4.0.1",
+    "aws-sdk-client-mock-jest": "4.0.1",
     "esbuild": "0.20.2",
     "eslint": "8.56.0",
     "eslint-config-prettier": "9.1.0",
@@ -55,8 +55,8 @@
     "uuid": "9.0.1"
   },
   "dependencies": {
-    "@aws-sdk/client-mwaa": "3.556.0",
-    "aws-cdk-lib": "2.139.0",
+    "@aws-sdk/client-mwaa": "3.609.0",
+    "aws-cdk-lib": "2.147.3",
     "axios": "1.6.8",
     "constructs": "10.3.0",
     "dotenv": "16.4.5"

--- a/usecases/start-stop-mwaa-environment/tsconfig.json
+++ b/usecases/start-stop-mwaa-environment/tsconfig.json
@@ -4,7 +4,7 @@
     "module": "CommonJs",
     "moduleResolution": "node",
     "lib": [
-      "ES2018"
+      "ESNext"
     ],
     "esModuleInterop": true,
     "skipLibCheck": true,

